src/storage/btree.c

```diff
@@ -12222,7 +12224,11 @@ btree_find_oid_from_ovfl (THREAD_ENTRY * thread_p, BTID_INT * btid_int, PAGE_PTR
 	      *offset_to_object = CAST_BUFLEN (oid_ptr - ovf_record.data);
 	    }
 	  /* Not in this page. */
```

Comment:
@razvanradulescu 

```C
          (void) btree_unpack_mvccinfo (ptr, mvcc_info, BTREE_OID_HAS_MVCC_INSID_AND_DELID);
          error_code = btree_find_oid_does_mvcc_info_match (thread_p, mvcc_info, purpose, match_mvccinfo, &is_match);
          if (error_code != NO_ERROR)
            { 
              ASSERT_ERROR ();
              return error_code;
            }
          if (is_match)
            {
              /* Object is a match. */
              *offset_to_object = CAST_BUFLEN (oid_ptr - ovf_record.data);
            }
          /* Not in this page. */
```

I think the above snippets are useless. Please confirm.